### PR TITLE
[ICDS Dashboard] Added fields to household ucr for bihar api

### PR DESCRIPTION
**Pull request by challabeehyv**
https://app.asana.com/0/0/1160919599791898/f

##### SUMMARY
Added fields to household ucr for bihar api

##### FEATURE FLAG
No

### CODE REVIEW

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left a few comments.

**Comment by Rohit25negi** [comment]
@calellowitz hh_minority and hh_caste is there in the person_Case ucr which is filled by referencing related household doc. I think its good to have these here as well to avoid the issues like we faced for dob property added child_health UCR which was being filled by referencing person case. What you think?

File: custom/icds_reports/ucr/data_sources/household_cases.json [hunk 1]
```diff
@@ -90,6 +90,86 @@
...
+        "is_primary_key": false,
+        "column_id": "hh_bpl_apl",
+        "is_nullable": true,
+        "type": "raw",
+        "property_name": "hh_bpl_apl"
+      },
+      {
+        "display_name": null,
+        "property_path": [],
+        "datatype": "boolean",
```
[the remark this answers is not in the document]
Comment (Rohit25negi) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: we dont have boolean type for ucr. we have small_integer using which we repsent 0/1

**Comment by Rohit25negi** [comment]
@challabeehyv you can pick this after you are done with the thing you are doing this is the highest priority for the dashboard team at the moment.

File: custom/icds_reports/ucr/data_sources/household_cases.json [hunk 2]
```diff
@@ -90,6 +90,86 @@
         "type": "raw",
         "property_name": "closed_on"
       },
+      {
+        "display_name": null,
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: done

File: custom/icds_reports/ucr/data_sources/household_cases.json [hunk 3]
```diff
@@ -90,6 +90,86 @@
...
+        "property_path": [],
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: done

File: custom/icds_reports/ucr/data_sources/household_cases.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: done

File: custom/icds_reports/ucr/data_sources/household_cases.json [hunk 4]
```diff
@@ -90,6 +90,86 @@
...
+        "is_primary_key": false,
+        "column_id": "hh_reg_date",
+        "is_nullable": true,
+        "type": "raw",
+        "property_name": "hh_reg_date"
+      },
+      {
+        "display_name": null,
+        "property_path": [],
+        "datatype": "string",
```
[the remark this answers is not in the document]
Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: done

File: custom/icds_reports/ucr/data_sources/household_cases.json [hunk 5]
```diff
@@ -90,6 +90,88 @@
         "type": "raw",
         "property_name": "closed_on"
       },
+      {
+        "datatype": "string",
+        "is_primary_key": false,
```
Comment (Rohit25negi) [inline custom/icds_reports/ucr/data_sources/household_cases.json]: you dont need `is_primary_key` and `is_nullable` key to add in every field.

**Review by Rohit25negi — APPROVED** [review]
Looks good to me. Left one comment. Not blocking but good to have.

File: custom/icds_reports/ucr/data_sources/household_cases.json (hunk 5 above)
Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: just added them from earlier columns..will remove them

Comment (challabeehyv) [reply custom/icds_reports/ucr/data_sources/household_cases.json]: removed

**Review by calellowitz — APPROVED** [review]

**Pull request merged**